### PR TITLE
Ded admin

### PR DESCRIPTION
**Pull request by robotmaxtron**

### CODE REVIEW

File: ansible/roles/openshift_dedicated_admin/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Example Playbook
+----------------
+
+    - hosts: servers
+      roles:
+         - role: openshift_dedicated_admin
+	   oda_ded_admin_users:
+	   - user1
+           - user2
```
Comment (twiest) [inline ansible/roles/openshift_dedicated_admin/README.md]: Spacing issue (should be left one space)

File: ansible/roles/openshift_dedicated_admin/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+- name: Setup the OpenShift Dedicated Admin service config file
+  template:
+    src: openshift-dedicated-role.j2
+    dest: /etc/sysconfig/openshift-dedicated-role
+    mode: "0644"
+    owner: root
+    group: root
+    notify:
+    - restart openshift-dedicated-role
```
Comment (twiest) [inline ansible/roles/openshift_dedicated_admin/tasks/main.yml]: Where is this handler implemented?

**Comment by robotmaxtron** [comment]
okay to test

**Comment by robotmaxtron** [comment]
okay to test

**Comment by robotmaxtron** [comment]
ok to test

File: ansible/roles/openshift_dedicated_admin/tasks/main.yml (hunk 2 above)
Comment (robotmaxtron) [inline ansible/roles/openshift_dedicated_admin/tasks/main.yml]: I've added a handlers section to the role in the handlers directory.

File: ansible/roles/openshift_dedicated_admin/handlers/main.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+# handlers file for openshift_dedicated_admin
+handlers:
+    - name: restart openshift-dedicated-role
+      service: 
+        name=openshift-dedicated-role 
+        state=restarted
```
Comment (twiest) [inline ansible/roles/openshift_dedicated_admin/handlers/main.yml]: We prefer using yaml syntax for this. So:

```
handlers:
- name: restart openshift-dedicated-role
  service: 
    name: openshift-dedicated-role 
    state: restarted
```

**Comment by twiest** [comment]
@robotmaxtron other than my minor comment, this is looking good. I'm ready to +1 this, once you've responded to my comment.

File: ansible/roles/openshift_dedicated_admin/handlers/main.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+# handlers file for openshift_dedicated_admin
+handlers:
+- name: restart openshift-dedicated-role
+  service: 
+    name=openshift-dedicated-role 
+    state=restarted
```
Comment (twiest) [inline ansible/roles/openshift_dedicated_admin/handlers/main.yml]: We prefer to use yaml syntax for everything possible. It makes the ansible error messages much better, and allows us to pass in complex arguments.

Consider changing this to:

```
handlers:
 - name: restart openshift-dedicated-role
   service: 
     name: openshift-dedicated-role 
     state: restarted
```

Comment (robotmaxtron) [inline ansible/roles/openshift_dedicated_admin/handlers/main.yml]: Okay, got it fixed now.

**Comment by twiest** [comment]
:+1:

**Pull request merged**